common-rendering/src/components/liveBlockContainer.tsx

```diff
@@ -86,9 +86,9 @@ const BlockByline = ({
 						css={css`
 							border-radius: 100%;
 							object-fit: cover;
-							height: 100%;
-							width: 100%;
-							background-color: ${avatarBackgroundColor};
+							height: 36px;
+							width: 36px;
```

Comment:
In AR, the byline image wasn't respecting the width and height of the container and was rendered as a 300px wide image. This was puzzling because it is a shared component and the behaviour in DCR was correct. Also, unticking and then ticking `height: 100%; width: 100%;` in the dev tools fixed the issue, which is very mysterious.
I am still not sure why this was happening - maybe some sort of race condition in AR? Anyway, passing an explicit size to the image fixes the problem.

Reply:
You may want `rem`s instead of px. This is `2.25rem`.